frontend/src/pages/projects/projectSettings/ModelBiasSettingsCard.tsx

```diff
@@ -52,7 +51,9 @@ const ModelBiasSettingsCard: React.FC<ModelBiasSettingsCardProps> = ({ namespace
           isLiveRegion
           isInline
         >
-          {error?.message}
+          {error?.message.includes('404')
```

Comment:
```suggestion
         // This is a temporary fix, this should be updated to incorporate work from https://github.com/opendatahub-io/odh-dashboard/pull/2032 in the future to provide a better experience.
          {error?.message.includes('404')
```

Reply:
Hey Pushpa, could you add the requested change above to include the comment? I think you should just be able to click "Commit suggestion" here. Thanks!